src/gui/HelpWindowController.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package gui;
+
+import java.net.URL;
+import java.util.ResourceBundle;
+import javafx.event.ActionEvent;
+import javafx.fxml.FXML;
+import javafx.fxml.Initializable;
+import javafx.scene.control.Button;
+import javafx.stage.Stage;
+
+/**
+ * FXML Controller class
+ *
+ * @author viktorianadarajah
+ */
+public class HelpWindowController extends Controller implements Initializable {
+    
+    private Stage stage;
+    @FXML
+    private Button closeButton;
```

Comment:
Doc block på attributter.

Reply:
fixed-in: 8db074ab0e284833c3be8adca8938e345183856a